<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaxWidget</class>
 <widget class="QWidget" name="FaxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="qss_tags" stdset="0">
   <stringlist notr="true">
    <string>xlet_frame</string>
   </stringlist>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="send_fax_label">
     <property name="text">
      <string>Send fax</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="qss_tags" stdset="0">
      <stringlist notr="true">
       <string>xlet_h1</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="step1_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="step1_label">
       <property name="text">
        <string>1.</string>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>step_number</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="choose_file_label">
       <property name="text">
        <string>Choose file to send</string>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_h4</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="file_picker_group">
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>input_button</string>
        </stringlist>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="FileNameLineEdit" name="file_name_input">
          <property name="placeholderText">
           <string>file</string>
          </property>
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="file_browse_button">
          <property name="text">
           <string>BROWSE...</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>black</string>
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="step2_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="step2_label">
       <property name="text">
        <string>2.</string>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>step_number</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="choose_number_label">
       <property name="text">
        <string>Choose destination number</string>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_h4</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="fax_number_group">
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>input_button</string>
        </stringlist>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="fax_number_input">
          <property name="placeholderText">
           <string>fax number</string>
          </property>
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fax_number_search_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../xivoclient.qrc">
            <normaloff>:/images/search-regular.svg</normaloff>:/images/search-regular.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="qss_tags" stdset="0">
           <stringlist notr="true">
            <string>black</string>
            <string>styled</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="send_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="send_fax_button">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>SEND FAX</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>orange</string>
         <string>styled</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_icon">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_text">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="xlet_p" stdset="0">
        <stringlist/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileNameLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">xletlib/extendedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>send_fax_button</tabstop>
 </tabstops>
 <resources>
  <include location="../../../xivoclient.qrc"/>
 </resources>
 <connections/>
</ui>
